docs/ERC-725.md

```diff
@@ -58,7 +58,7 @@ The `operationType` should represent the assembly operation as follows:
 - `1` for `delegatecall`
 - `2` for `create`
 
-Others may be added in the future. Inspired by [ERC1077](https://eips.ethereum.org/EIPS/eip-1077) and [Gnosis](https://github.com/gnosis/safe-contracts/blob/master/contracts/Enum.sol#L7)
+Others may be added in the future. Inspired by [ERC1077](https://eips.ethereum.org/EIPS/eip-1077) and [Gnosis](https://github.com/gnosis/safe-contracts/)
```

Comment:
How about linking to this file: https://github.com/gnosis/safe-contracts/blob/master/contracts/common/Enum.sol